doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
Btw, I still feel like the documentation change (the deprecation) is semver-major.

The js part looks semver-minor.